environments/aat.yml

```diff
@@ -158,6 +158,12 @@ cname:
   - name: "immigration-appeal"
     ttl: 3600
     record: "hmcts-aat.azurefd.net"
+  - name: "afdverify.ia-wa-proto-frontend"
+    ttl: 3600
+    record: "afdverify.hmcts-aat.azurefd.net"
+  - name: "ia-wa-proto-frontend"
```

Comment:
shouldn't need this in public dns, add it to hmcts/azure-private-dns

Reply:
done. Ta